source/Applications/PointCloudTutorial.md

```diff
@@ -0,0 +1,131 @@
+
+## Introduction
+
+This tutorial describes how to use Zivid SDK to work with [Point Cloud][kb-point_cloud-url] data.
+
+1. [Frame](#frame)
+   1. [Capture](#capture)
+   2. [Load](#load)
+   3. [Visualize](#visualize)
+3.  [Point Cloud](#point-cloud)
+    1. [Get](#get)
+    2. [Copy](#copy)
+    3.  [Transform](#transform)
+
+### Prerequisites
+
+You should have installed Zivid SDK and cloned C++ samples. For more details see [Instructions][installation-instructions-url].
+
+## Frame
+
+### Capture
+
+The ```Zivid::Frame``` can be captured with a camera ([go to source][frame-capture]).
+```cpp
+const auto frame = camera.capture(settings);
+```
+Check [Capture Tutorial][capture-tutorial] for more details.
+
+### Load
+It can also be loaded from a ZDF file ([go to source][frame-from-file]).
+```cpp
+const Zivid::Frame frame = Zivid::Frame(Zivid3D.zdf);
+```
+### Visualize
+
+Having the frame allows you to visualize the point cloud ([go to source][visualize-point-cloud]).
+
+```cpp
+Zivid::Visualization::Visualizer visualizer;
+
+visualizer.showMaximized();
+visualizer.show(frame);
+visualizer.resetToFit();
+
+visualizer.run();
+```
+
+## Point Cloud
+
+### Get
+
+You can now get a handle to the point cloud data on the GPU ([go to source][point-cloud]).
+```cpp
+frame.pointCloud();
+```
+Note: ```Zivid::Camera::capture()``` method returns as soon as the camera is done capturing. However, the point cloud becomes available only after the GPU is finished processing. Therefore, if you call ```Zivid::Frame::pointCloud()``` right after the capture, the CPU will wait until the point cloud is available and then return.
+
```

Comment:
The first sentence is accurate. However, the rest is inaccurate, probably because I said something wrong to you earlier.

`Frame::pointCloud()` always returns instantly. Any wait-time for processing occurs when you start calling `pointCloud.copyXXXX()` functions. Perhaps it can be something like this:

"
Note: ```Zivid::Camera::capture()``` method returns as soon as the camera is done capturing. At that point the handle from `frame.pointCloud()` is available instantly as well. However, the actual point cloud data becomes available only after the GPU is finished processing. This processing will occur on the GPU in the background, and any calls to data-copy functions (see section "Copy" below) will wait for this to finish before proceeding with the requested copy operation.
"

Reply:
Done, thanks for the suggestion. Used it.